Dockerfile

```
@@ -66,5 +66,9 @@ COPY --from=builder /zipline/node_modules ./node_modules
 COPY --from=builder /zipline/node_modules/.prisma/client ./node_modules/.prisma/client
 COPY --from=builder /zipline/node_modules/@prisma/client ./node_modules/@prisma/client
 
-# Run the application
-CMD ["node", "--enable-source-maps", "dist/index.js"]
\ No newline at end of file
+# Copy Startup Script
+COPY entrypoint.sh /usr/local/bin/docker-entrypoint.sh
```

```suggestion
COPY docker-entrypoint.sh /zipline
```

Should be `docker-entrypoint.sh` instead of `entrypoint.sh`, and lets just move it into /zipline.